File: Technical/Python/Level-0/Conditonal Statements/Days.py
# Q - 6 : Print Days
# Question Type: Coding
# Problem Statement
# Write a program to print corresponding day based on given input

# Input Format
# Accept an integer as input

# Output Format
# Print corresponding day or invalid

# Constraints
# NA
day = int(input(""))
if day == 0:
    print("Sunday")
elif day == 1:
    print("Monday")
elif day == 2:
    print("Tuesday")
elif day == 3:
    print("Wednesday")
elif day == 4:
    print("Thursday")
elif day == 5:
    print("Friday")
elif day == 6:
    print("Saturday")
else:
    print("Invalid")

File: Technical/C Programming/Level 0/Conditional Statements/Days.c
/*
Q-8 : Days
Question Type: Coding
Problem Statement: Write a program to print corresponding day based on given input
Input Format: Accept integer as a input
Output Format: Print corresponding day or invalid
Constraints: NA
*/
#include<stdio.h>
main()
{
	int d;
	scanf("%d",&d);
	switch(d)
	{
		case 0:
		 printf("Sunday");
		 break;
		case 1:
		 printf("Monday");
		 break;
		case 2:
		 printf("Tuesday");
		 break;
		case 3:
		 printf("Wednesday");
		 break;
		case 4:
		 printf("Thursday");
		 break;
		case 5:
		 printf("Friday");
		case 6:
		 printf("Saturday");
		 break;
		default:
		 printf("Invalid");
	}
}
